projects/packages/videopress/src/tus/class-tus-file.php

```diff
@@ -106,6 +106,20 @@ public function __construct( $name = null, $cache = null ) {
 		$this->cache = $cache;
 	}
 
+	/**
+	 * Returns an integer if it's castable. Otherwise it throws
+	 *
+	 * @param string|int $number Number to cast.
+	 * @throws InvalidArgumentException If argument is invalid.
+	 * @return int
+	 */
+	public function ensure_integer( $number ) {
+		if ( ! is_numeric( $number ) ) {
+			throw new InvalidArgumentException( 'argument needs to be an integer. Check stacktrace' );
```

Comment:
Personally I might pass in the argument name to include in the exception message, but I won't insist.

Reply:
Was about to do it but then realized it gets to be noisy in the code while it will still be as noisy in the logs